linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -144,6 +144,10 @@ impl ClientWrapper {
         let mut command = Command::new(path);
         command
             .current_dir(self.path_provider.path())
+            .env(
+                "RUST_LOG",
+                std::env::var("RUST_LOG").unwrap_or(String::from("linera=default")),
```

Comment:
I'm seeing this now in end-to-end tests:

```
ignoring `linera=default`: invalid filter directive
```

Should we just not set `RUST_LOG` instead? Or did you mean `linera=debug`?